### DISCUSSION TITLE
holes in point grid at fine spatial resolution

### DISCUSSION THREAD

**Discussion by hardreddata** [insarlab/MintPy#535] [Q&A] [answered]
Hi,

I am enjoying exploring MintPy. Thanks again for sharing it. 

It sounds like an exciting time for SAR with ICEYE and Capella Space entering the space.

Suppose I am trying to get the most resolution I can out of Sentinel. 

The ISCE `stackSentinel.py` arguments after the data, orbit, aux, DEM and bounding box.

```
-r 3 -z 1 -f 0.5 -C geometry -u icu --start_date 2021-01-01
```

And then doing something very minimal with MintPy, after loading up the ISCE stack per documentation.

```
mintpy.networkInversion.weightFunc      = no #[var / fim / coh / no], auto for var 
mintpy.topographicResidual              = no  #[yes / no], auto for yes
```

Creating a shapefile `save_qgis.py geo/geo_timeseries_ERA5.h5 -g geo/geo_geometryRadar.h5` I note the points are often chunked into in strips

![image](https://user-images.githubusercontent.com/42530666/111242580-8bc0ad80-8653-11eb-8928-999cff787dca.png)

Here this is a toy example with only three snapshots. I see the same thing when looking at years of data with more thoughtful parameters (NESD, Snaphu, MintPy network modification etc).

I wonder if this is because the SRTM DEM is only 30m? Or perhaps ISCE or MintPy isn't intended for such a small number of range and azimuth looks? I want to explore replacing ISCE with Fringe but have had issues https://github.com/isce-framework/fringe/issues/41.

Advice invited.

**Comment by piyushrpt** [comment, marked as answer]
If you have a stack in radar geometry, you are much better of generating the point cloud from that rather than from geocoded results. Geocoding inherently will force it on to a 30m grid. If you are looking for resolution, the radar geometry lat/lon should be better. Coarser azimuth resolution will dominate and you will still see linear patterns with S1 data.

**Comment by yunjunz** [comment]
By default, mintpy is geocoding in roughly the same grid size as the input file in radar coordinates ([resample.py#L327](https://github.com/insarlab/MintPy/blob/main/mintpy/objects/resample.py#L327)) using the nearest-neighbor interpolation. I second @piyushrpt recommendation on feeding the radar-coord files to `save_qgis.py` directly, especially for the high-resolution point cloud as your case.

**Comment by hardreddata** [comment]
Thanks!

I didn't strictly realise I could combine inputs and outputs in the `save_qgis.py` command. So many MintPy tricks to learn.

It looks much nicer now.
